v3/vertex/angle65/angle65.JSON

```diff
@@ -0,0 +1,193 @@
+{
+  "name": "ANGLE65",
+  "vendorId": "0x9954",
+  "productId": "0x9970",
```

Comment:
VID and PID incorrect with QMK ones

Reply:
I have modified duplicate VID/PID https://github.com/qmk/qmk_firmware/pull/20401